kolibri/core/templates/kolibri/base.html

```diff
@@ -3,7 +3,7 @@
 {% get_current_language_bidi as LANGUAGE_BIDI %}
 {% get_current_language as LANGUAGE_CODE %}
 <!DOCTYPE html>
-<html dir="{{ LANGUAGE_BIDI|yesno:'rtl,ltr' }}">
+<html dir="{{ LANGUAGE_BIDI|yesno:'rtl,ltr' }}" lang="{{ LANGUAGE_CODE }}">
```

Comment:
this is cool, but how did it end up in this PR?

Reply:
I was already modifying this file, so I said might as well. `:slightly_smiling_face:`